src/mono/netcore/System.Private.CoreLib/src/ILLink/ILLink.LinkAttributes.wasm.xml

```diff
@@ -0,0 +1,133 @@
+<linker>
+  <assembly fullname="*">
```

Comment:
Can we limit the assembly "*" to only the cases where it's necessary? it's relatively expensive for the linker (as it has to search all assemblies in the closure and find all occurrences of the type).

For most of these attributes we know the assembly they come from, so we should be specific.

I personally would also vote for adding the XML to the same assembly the attribute is from (just so that we don't have unnecessary cross assembly references, especially the "wrong" way from CoreLib upwards).

Reply:
yeah, we could split the types which are not compiler-generated into its own section

Reply:
Done.